icons.go

```diff
@@ -12,8 +12,10 @@ type AppData interface {
 
 // ApplicationProvider describes a type that can locate icons and applications for the current system
 type ApplicationProvider interface {
+	AllApps() []AppData
+	AvailableThemes() []string
 	FindAppFromName(appName string) AppData
 	FindAppFromWinInfo(win Window) AppData
 	FindAppsMatching(pattern string) []AppData
-	DefaultApps() []AppData
```

Comment:
I didn't imagine that we should down-cast our typed API to just strings - the user of this interface would surely expect the same type from DefaultApps as AllApps?